.golangci.yml

```diff
@@ -1,9 +1,21 @@
 linters:
   enable:
     - errcheck
+    - errname
+    - errorlint
+    - goconst
     - gofumpt
     - gosimple
     - govet
     - ineffassign
+    - nilerr
+    - prealloc
+    - predeclared
+    - revive
+    - rowserrcheck
+    - sqlclosecheck
     - staticcheck
+    - unconvert
     - unused
+    - usestdlibvars
+    - wastedassign
```

Comment:
Revive is a powerful linter, but people often uses only the default checkers available

Here I'm activating a few linters that I checked to provide errors on your project

```suggestion
    - wastedassign
    
 linters-settings:
revive:
    rules:
      # these are the default revive rules
      - name: blank-imports
      - name: context-as-argument
        arguments:
          - allowTypesBefore: "*testing.T"
      - name: context-keys-type
      - name: dot-imports
      - name: empty-block
      - name: error-naming
      - name: error-return
      - name: error-strings
      - name: errorf
      - name: increment-decrement
      - name: indent-error-flow
      - name: range
      - name: receiver-naming
      - name: redefines-builtin-id
      - name: superfluous-else
      - name: time-naming
      - name: unexported-return
      - name: unreachable-code
      - name: unused-parameter
      - name: var-declaration
      - name: var-naming
      
      # now the linters that could be enabled
      - name: unnecessary-stmt
      - name: deep-exit
      - name: max-control-nesting
      - name: confusing-naming
      - name: unused-receiver
      - name: unhandled-error
```

Reply:
Thanks! I'll incorporate them.